pkg/local_object_storage/blobovnicza/blobovnicza.go

```diff
@@ -0,0 +1,103 @@
+package blobovnicza
+
+import (
+	"os"
+
+	"github.com/nspcc-dev/neofs-node/pkg/util/logger"
+	"go.etcd.io/bbolt"
+	"go.uber.org/atomic"
+	"go.uber.org/zap"
+)
+
+// Blobovnicza represents the implementation of NeoFS Blobovnicza.
+type Blobovnicza struct {
+	*cfg
+
+	filled *atomic.Uint64
+
+	boltDB *bbolt.DB
+}
+
+// Option is an option of Blobovnicza's constructor.
+type Option func(*cfg)
+
+type cfg struct {
+	boltDBCfg
+
+	fullSizeLimit uint64
+
+	objSizeLimit uint64
+
+	log *logger.Logger
+}
+
+type boltDBCfg struct {
+	perm os.FileMode
+
+	path string
+
+	boltOptions *bbolt.Options
```

Comment:
`boltOptions` has no set references in the code. Is it correct?

Reply:
I planned to add option to specify DB options, but forgot, thx.